WordPress/Classes/Services/BlogService.m

```diff
@@ -108,12 +126,15 @@ - (Blog *)lastUsedBlog
         return nil;
     }
 
-    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:@"Blog"];
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"visible = YES AND url = %@", url];
+    NSFetchRequest *fetchRequest = [NSFetchRequest fetchRequestWithEntityName:BlogEntity];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%@ AND url = %@", IsVisiblePredicate, url];
```

Comment:
Creating consts for NSStrings was great :), but in the case of `IsVisiblePrediate` and `IsWPComAndVisiblePredicate` it would be nice to keep them inline so you don't have to check multiple places to see exactly what the predicate does.